core_impl/src/test/java/com/google/instrumentation/stats/StatsContextFactoryTest.java

```diff
@@ -29,14 +31,19 @@
  */
 @RunWith(JUnit4.class)
 public class StatsContextFactoryTest {
```

Comment:
It looks like there aren't any tests that directly test the output from the serialize method.  I think there should also be some tests involving multiple tags.